src/App.css

```
@@ -1,42 +1,181 @@
+@tailwind base;
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)